manifests/l/LOOT/LOOT/0.20.0/LOOT.LOOT.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LOOT.LOOT
+PackageVersion: 0.20.0
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{BF634210-A0D4-443F-A657-0DCE38040374}_is1'
+ReleaseDate: 2023-06-10
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/loot/loot/releases/download/0.20.0/loot_0.20.0-win64.exe
+  InstallerSha256: FCB8D22F14D45D716F622D6A79337FD5CAD5486447133BA4A13253EF0D08A183

```

Comment:
```suggestion
```
It appears that [the `0.20.0` release](https://github.com/loot/loot/releases/tag/0.20.0) no longer includes `x86` support: 
> LOOT requires a 64-bit version of Windows 10 (1809) or later and the [MSVC 2019 x64 redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe).